components/ShowJobInformation.js

```diff
@@ -20,10 +20,10 @@ const ShowJobInformation = (props) => {
     },
 
     cardContainer: {
-      display: "flex",
       justifyContent: "center",
       alignContent: "center",
```

Comment:
If `display: "flex"` is not used, then `justifyContent` and `alignContent` properties are stale as well.